SPECIFICATION.md

```diff
@@ -1443,6 +1443,11 @@ The Use-API POJOs can also expose a public method, called `init`, with the follo
 The `bindings` map can contain objects that provide context to the currently executed HTL script that the Use-API object can use for its processing.
 
 ### 4.2. JavaScript Use-API
+
+>The JavaScript Use API has been deprecated for use with AEM as a Cloud Service. Please use [the Java Use API instead.](https://experienceleague.adobe.com/en/docs/experience-manager-htl/content/java-use-ap)
```

Comment:
The previous link was returning a 404 for me, the one from the suggestion takes me to the correct place.

```suggestion
>The JavaScript Use API has been deprecated for use with AEM as a Cloud Service. Please use [the Java Use API instead.](https://experienceleague.adobe.com/en/docs/experience-manager-htl/content/java-use-api)
```

Reply:
Copy paste error. Thanks for catching. LGTM